DuggaSys/microservices/monolithic-to-microservices.md

```diff
@@ -0,0 +1,123 @@
+
+Disclaimer, this guide serves as a guideline and can not be followed 100% since the structure of all files differ. It's recommended to test after each step which include code changing to so its faster to spot if something goes wrong. When this guide was written no service using retrive information was used, write guide at step 5.
+***
+##Setting up file & folder:
```

Comment:
I think there are spaces missing after the '#' characters

Reply:
Spaces makes no difference after ###